zenith_utils/src/zid.rs

```diff
@@ -221,6 +221,10 @@ impl fmt::Display for ZTenantTimelineId {
     }
 }
 
+// Unique ID of a storage node (safekeeper or pageserver). Supposed to be
+// sequentially issued by the console.
+pub type ZNodeId = u64;
```

Comment:
Lets make it an actual newtype, because this way we can safely assign any u64 to a variable of type ZNodeId

Reply:
You mean you want the assignment look like `node_id.0 = x` instead of `node_id = x` to prevent accidental assignment?

Reply:
I mean e g when function parameter or a struct variable expects type that is an alias it accepts the original type that alias points to. See the difference here: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=e64599c52d153d3b627d95ca7653ab24

And thats why current ZId derivatives are newtypes instead of aliases, that avoids possible confusion with plain ZId type. So you cannot pass ZId to function that expects ZTenantId

Reply:
Ah, ok, thanks.